SU-SWS/stanford_profile_helper
# D8CORE-2322: fixing the margins on the full width page

# READY FOR REVIEW

# Summary
- Fixing the margins for full width page

# Needed By (Date)
- 8/17

# Urgency
- Important

# Steps to Test

1. Pull in this change
2. Check the margins on the homepage and see that the xl is 100px. Make sure the full width banners are still full width. 

# Affected Projects or Products
- Core and SOE

# Associated Issues and/or People
- D8CORE-2322
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
